### ISSUE TITLE
On PHP documents, code completion for element attributes "class" and "style" in mixed HTML lines do not work

### ISSUE THREAD

**Issue by outofsync42** [microsoft/vscode#158020]
On PHP documents, code completion for element attributes "class" and "style" in mixed HTML lines only works if you set 

editor.quickSuggestions[strings]= true

```
"[php]": {
    "editor.quickSuggestions": {
	    "strings": true //required for <html class="" style="">
    },
},
```

class and style attributes appear to be read as strings and while this hack works it then forces me to have all string through out the PHP document to have code completion when i only want it for class and style attributes.

**Comment by roblourens** [comment]
HTML features don't work well in php files

**Comment by outofsync42** [comment]
I'm aware of that. I needed to write an extension of my own just to compensate for lack of support for "html.autoClosingTags" and "html.autoCreateQuotes" settings not working. While I was able to fix those issues, I am unable to compensate for code completion issues because `vscode.commands.executeCommand('vscode.executeCompletionItemProvider',url,position)` command does nothing when called while the cursor is inside of an html attribute. If i could at least call it manually i would be fine but it does not seem to work.
